lib/sdr_client/deposit/file.rb

```diff
@@ -6,13 +6,14 @@ module Deposit
     class File
       # rubocop:disable Metrics/ParameterLists
       def initialize(external_identifier:, label:, filename:,
-                     access: 'world', preserve: true, shelve: true,
+                     access: 'dark', download: 'none', preserve: true, shelve: true,
```

Comment:
wow - it will be interesting to see what this change of default rights does to everything ...